ci_framework/roles/edpm_deploy/tasks/main.yml

```diff
@@ -108,3 +127,21 @@
   ansible.builtin.include_role:
     name: 'install_yamls_makes'
     tasks_from: 'make_edpm_deploy_instance'
+
+- name: Run gmake command manually
+  environment:
+    PATH: "{{ cifmw_path }}"
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+  ansible.builtin.shell: |
```

Comment:
ci_make would allow to get the outputs in files directly. I don't see any benefit of running via `shell` here.